lv2/core/lv2.h

```diff
@@ -366,7 +371,9 @@ typedef struct LV2_Descriptor {
    Put this (LV2_SYMBOL_EXPORT) before any functions that are to be loaded
    by the host as a symbol from the dynamic library.
 */
-#ifdef _WIN32
+#if defined(__EMSCRIPTEN__)
+#  define LV2_SYMBOL_EXPORT LV2_SYMBOL_EXTERN EMSCRIPTEN_KEEPALIVE
```

Comment:
This appears to just be defined as `__attribute__((used))`.  I'd prefer that to `lv2.h` including `emscripten.h` (which is quite a monster).

Reply:
yeah that seems like a sensible idea